packages/hint-validate-summary/tests/tests.ts

```diff
@@ -0,0 +1,78 @@
+import { test, fs } from '@hint/utils';
+import { HintTest, testHint } from '@hint/utils-tests-helpers';
+
+const { generateHTMLPage, getHintPath } = test;
+const hintPath = getHintPath(__filename);
+
+const { readFile } = fs;
+
+
+const generateConfig = (fileName: string) => {
+
+
+    const styles = readFile(`${__dirname}/fixtures/${fileName}.css`);
+
+    return {
+        '/': generateHTMLPage(`<link rel="stylesheet" href="${fileName}.css">`),
+        [`/${fileName}.css`]: {
+            content: styles,
+            headers: { 'Content-Type': 'text/css' }
+        }
+    };
+};
+
+const summary = {
+    summaryDefaultStyle: generateConfig('summaryDefaultStyle'),
+    summaryStyleChanged: generateConfig('summaryStyleChanged'),
+    summaryTest1: generateConfig('summaryTest1'),
+    summaryTest2: generateConfig('summaryTest2'),
+    summaryTest3: generateConfig('summaryTest3'),
+    summaryTest4: generateConfig('summaryTest4'),
+    summaryTest5: generateConfig('summaryTest5'),
+    summaryTest6: generateConfig('summaryTest6')
```

Comment:
I'd recommend also including a test for a `summary` rule that doesn't specify `display` at all (which should pass).